ruby-challenges.rb

```diff
@@ -19,11 +28,29 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+# psuedocode:
+# 1. Create a method
+# 2. Take in an array of numbers
+# 3. Return the sum of the numbers
+
+
+def sum_nums(array)
+  array.reduce(:+)
+end
```

Comment:
Cool approach!  Ruby also has a built in method called sum that will also perform the same action and oftentimes a little faster.